client/src/pages/ProductPage/ProductPage.tsx

```diff
@@ -1,46 +1,79 @@
 import style from './styles.module.scss';
-import img1 from '@/assets/product_p.png';
-import button_up from '@/assets/button_up.svg';
-import button_down from '@/assets/button_down.svg';
+import img1 from '@/assets/sample1.jpg';
+import img2 from '@/assets/sample2.jpg';
 import { ProductsDescription } from '@/components/ProductDescription/ProductsDescription';
 import { BottomDescription } from '@/components/BottomDescription/BottomDescription';
 import { SectionTitle } from '@/shared/components/SectionTitle/SectionTitle';
 import { Products } from '@/components/Products/Products';
 import { useAppSelector } from '@/app/hooks';
 import { RootState } from '@/app/store';
 import { useParams } from 'react-router-dom';
+import { useState, useEffect } from 'react';
+import clsx from 'clsx';
 
 export const ProductPage = () => {
   const { id: idString } = useParams();
-
   const id = Number(idString);
-
   const { products } = useAppSelector((state: RootState) => state.products);
 
+  const [selectedImg, setSelectedImg] = useState<string | undefined>(undefined);
+
+  useEffect(() => {
+    if (products.length > 0) {
```

Comment:
for readability it is common to use early return condition

if(products.length === 0) {
  return
}

... do the rest